bindings/matlab/+iDynTreeWrappers/updateVisualization.m

```diff
@@ -0,0 +1,27 @@
+function []=updateVisualization(KinDynModel,linkNames,transforms)
```

Comment:
I see there are not debug checks in the wrappers but maybe we can add them later (if necessary), based on the users feedback.

Reply:
Which kind of debug things would you add?

Reply:
For example, on the previous visualizer based on `irrlicht` sometimes the robot was not visualized correctly (I saw a white window), but the code was not stopping and the error message was only on the Matlab terminal or absent (e.g. I had this issue when using `softwareopengl` option in Matlab). I would say that we need to be sure that in case something in the visualization fails, it is clear for the user why the robot is not visualized correctly. But said this, it is too early now and  we need to first use the visualizer in order to understand which error messages may help the user.